pkgs/build-support/rust/cargo-vendor/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, callPackage, defaultCrateOverrides, fetchFromGitHub,
+  pkgconfig, zlib, openssl, libssh2, curl, libgit2,
+  darwin, libiconv }:
+
+let
+  inherit (darwin.apple_sdk.frameworks) CoreFoundation;
+in
+
+(callPackage ./cargo-vendor.nix {}).cargo_vendor_0_1_13.override {
```

Comment:
This can indeed be used in this way, but the intention is to gather all possible crateOverrides in pkgs/build-support/rust/default-crate-overrides.nix instead. The goal is that other users of the same packages won't have to do this again.

Reply:
Agreed.

Reply:
This mechanism is indented to overrides outside of nixpkgs.